packages/grid/x-data-grid/src/DataGrid/DataGrid.tsx

```diff
@@ -165,6 +165,12 @@ DataGridRaw.propTypes = {
    * @ignore - do not document
    */
   disableEval: PropTypes.bool,
+  /**
+   * If `true`, multiple selection using the Ctrl/CMD or Shift key is disabled.
+   * The MIT DataGrid will ignore this prop, unless `checkboxSelection` is enabled.
+   * @default false (`true` for MIT DataGrid)
```

Comment:
> (`true` for MIT DataGrid)

If that is the case, shouldn't [this demo](https://deploy-preview-11448--material-ui-x.netlify.app/x/react-data-grid/row-selection/#checkbox-selection) disallow multiple selection?
Seems like it's false for MIT Data Grid too. 🤔

Reply:
The default value for MIT Data Grid is:
- `false` if `checkboxSelection={true}`
- `true` if `checkboxSelection={false}`

I'll update the JSDoc to clarify this:

```
@default false (`true` for MIT DataGrid with `checkboxSelection={false}`)
```